server/tests/batch_comparison/test_batch_comparison.py

```diff
@@ -167,7 +167,7 @@ def test_batch_comparison_round_1(
     assert jurisdictions[1]["currentRoundStatus"] is None
 
     # Use an artificially large sample size in order to have enough samples to work with
-    sample_size = 20
+    sample_size = 10
```

Comment:
i guess this failed the validation? i think we needed that many samples for the rest of the test to be meaningful (e.g. where we check below how the batches got divvied up between the audit boards - with a small sample that check doesn't really check anything)

maybe instead of lowering the sample size, we could change the manifest so the validation passes?

Reply:
Ah, gotcha. Good point. What tweak would you recommend on the manifest so the math is happy?

Reply:
I'm not totally sure, but I would try out increasing the number of ballots until the sample size validation passes. I would recommend removing the `manifests` fixture from this test case's arguments and just copying the code in that fixture into this test case, then tweaking it.